### PR TITLE
refactor(tier4_control_launch): organized arguments

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

organize arguments and yaml in tier4_control_launch

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_control_launch/launch/control.launch.py [hunk 1]
```diff
@@ -174,11 +176,6 @@ def launch_setup(context, *args, **kwargs):
         parameters=[
             vehicle_cmd_gate_param,
             vehicle_info_param,
-            {
-                "use_emergency_handling": LaunchConfiguration("use_emergency_handling"),
-                "use_external_emergency_stop": LaunchConfiguration("use_external_emergency_stop"),
-                "use_start_request": LaunchConfiguration("use_start_request"),
-            },
```
Comment (takayuki5168) [inline launch/tier4_control_launch/launch/control.launch.py:181]: @isamu-takagi Do you wanna designate these variables not by a yaml file but by ros2 launch command?

**Comment by takayuki5168** [comment]
This PR must be merged after https://github.com/autowarefoundation/autoware_launch/pull/74

File: launch/tier4_control_launch/launch/control.launch.py (hunk 1 above)
Comment (isamu-takagi) [reply launch/tier4_control_launch/launch/control.launch.py:181]: I don't think it will be used from a command, so a file is OK.

Comment (takayuki5168) [reply launch/tier4_control_launch/launch/control.launch.py:181]: Thanks

**Comment by shmpwk** [comment]
LGTM

**Pull request closed without merging**